src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs

```diff
@@ -103,7 +102,7 @@ static async Task RunAsync(ServerConfiguration serverConfiguration, Cancellation
 
     // LSP server doesn't have the pieces yet to support 'balanced' mode for source-generators.  Hardcode us to
     // 'automatic' for now.
-    var globalOptionService = exportProvider.GetExportedValue<IGlobalOptionService>();
+    var globalOptionService = exportProvider.GetExportedValue<Microsoft.CodeAnalysis.Options.IGlobalOptionService>();
```

Comment:
After removing the `Cli` prefix (to make sure we don't break every single System.CommandLine user), `Microsoft.CodeAnalysis.Options.Option<T>` conflicts with `System.CommandLine.Option<T>`. I decided to remove the `using Microsoft.CodeAnalysis.Options` line and specify full name here as other solutions would require more changes.